client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -66,28 +66,39 @@ class StatsPeriodNavigation extends PureComponent {
 		const nextDayQuery = qs.stringify( Object.assign( {}, queryParams, { startDate: nextDay } ), {
 			addQueryPrefix: true,
 		} );
+		const isNextDisabled = hideNextArrow || isToday;
 
 		return (
 			<div className="stats-period-navigation">
 				<div className="stats-period-navigation__children">{ children }</div>
-				<a
-					className={ classNames( 'stats-period-navigation__previous', {
-						'is-disabled': hidePreviousArrow,
-					} ) }
-					href={ `${ url }${ previousDayQuery }` }
-					onClick={ this.handleClickPrevious }
-				>
-					<Icon className="gridicon" icon={ arrowLeft } />
-				</a>
-				<a
-					className={ classNames( 'stats-period-navigation__next', {
-						'is-disabled': hideNextArrow || isToday,
-					} ) }
-					href={ `${ url }${ nextDayQuery }` }
-					onClick={ this.handleClickNext }
-				>
-					<Icon className="gridicon" icon={ arrowRight } />
-				</a>
+
+				{ ! hidePreviousArrow ? (
+					<a
+						className={ classNames( 'stats-period-navigation__previous' ) }
+						href={ `${ url }${ previousDayQuery }` }
+						onClick={ this.handleClickPrevious }
+					>
+						<Icon className="gridicon" icon={ arrowLeft } />
+					</a>
+				) : (
+					<div className={ classNames( 'stats-period-navigation__previous', 'is-disabled' ) }>
+						<Icon className="gridicon" icon={ arrowLeft } />
+					</div>
+				) }
```

Comment:
Would it be possible to keep the element structure (`a.stats-period-navigation__previous > Icon.gridicon`) but conditionally unset the onClick handler to reduce the amount of duplicated code here?

```suggestion
				<a
					className={ classNames( 'stats-period-navigation__previous', {
						[ hidePreviousArrow ]: 'is-disabled',
					} ) }
					href={ `${ url }${ previousDayQuery }` }
					onClick={ hidePreviousArrow ? undefined : this.handleClickPrevious }
				>
					<Icon className="gridicon" icon={ arrowLeft } />
				</a>
```

Reply:
This also needs to remove `href` because that's what makes unwanted keyboard navigation possible. I will have to keep an anchor tag without both `onClick` and `href` if we want to keep the same HTML.